unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ExploreV2/Scripts/MainMenu/ExploreV2Menu/ExploreV2MenuComponentView.cs

```diff
@@ -291,4 +295,17 @@ internal RealmSelectorComponentView ConfigureRealmSelectorModal()
 
         return realmSelectorView;
     }
+
+    private void OnWorldChange(bool isWorld, bool wasWorld)
+    {
+        if (isWorld == wasWorld) return;
+
+        if (isWorld && IsSectionActive(ExploreSection.Map))
+        {
+            placesAndEventsSection.Show();
+            mapSection.Hide();
+        }
+
+        SetSectionActive(ExploreSection.Map, !isWorld);
+    }
```

Comment:
I'm wondering if this logic shoulnd't be in the controller side. The views shouldn't have the responsibility of knowing when something related to the world is happening. In my opinion, ideally, the views should only accept orders from controllers or from the UI controls of the own view.

Reply:
ok, I moved logic into controller in the new commit